file_transfer_protocol/client.py

```diff
@@ -0,0 +1,23 @@
+import socket                   # Import socket module
+
+sock = socket.socket()             # Create a socket object
+host = socket.gethostname()     # Get local machine name
+port = 12312
+
+sock.connect((host, port))
+sock.send(b'Hello server!')
+
+with open('Received_file', 'wb') as in_file:
```

Comment:
We write data to an out_file.

Reply:
sorry, I was confused, I was thinking it was sending it out (like a physical thing)